gameoflife/src/de/javacourse/ConsoleView.java

```diff
@@ -0,0 +1,27 @@
+package de.javacourse;
+
+public class ConsoleView
+{
+    public void printBoardToConsole(Board board, int rows, int cols)
+    {
+        Cell[][] cells = board.getCells();
```

Comment:
Grundsätzlich gibt das Board hier eine Referenz des Cell-Arrays raus. Damit wäre es möglich das Array von außerhalb (außerhalb von Board) zu verändern. Das will man in der Regel nicht. 
Sollten wir in der Stunde mal über Gründe dafür reden und wie man seine "Member" besser schützen kann.